### DISCUSSION TITLE
Editor for Lexicon LXP-1

### DISCUSSION THREAD

**Discussion by Tonfisch** [RomanKubiak/ctrlr#615] [Show and tell]
Hello Ctrlr's
here is the next one.

![grafik](https://github.com/RomanKubiak/ctrlr/assets/115865850/f6215c86-0d96-4595-81ae-640947c60ab1)

https://github.com/Tonfisch/LXP-1-Editor
Thanks to John Goodland, for his Help.

**Comment by dobo365** [comment]
Thx for this.
I was just busy chatting with someone else about Spiffo's PCM70 and PCM80 editors.
I have a MPX-1 and was thinking to use them as base for a MPX-1 editor.
Unfortunately I have only pictures.
If someone has the original panel or bpanelz file, thx for sharing.

**Reply by Tonfisch** [reply]
[PCM80 Editor v1.6.zip](https://github.com/RomanKubiak/ctrlr/files/14551367/PCM80.Editor.v1.6.zip)
[PCM70 Editor v1.3.zip](https://github.com/RomanKubiak/ctrlr/files/14551368/PCM70.Editor.v1.3.zip)
The PCM80 freeses after some time, for some time. I did not manage to catch the  reasons for that.

**Reply by Tonfisch** [reply]
maby it has to do with the autosave-funktion in Ctrlr-preferences.

**Reply by dobo365** [reply]
Thx a lot! Much appreciated!
I only wanted them as reference to see how they have been built.
Didn't dive in the MPX-1 panel design yet...

**Reply by dnaldoog** [reply]
Looks great- I nearly bought one, but got a Sony R7 instead, which is very nice - the sysex in the R7 is also packed in an arcane way. I finally worked out how though, but a panel for it will probably be as complex as this one looks.

**Reply by Tonfisch** [reply]
The Sony is also on my wishlist of things to get. They have a good reputation and i want to know how  and what they did, to get to their goals.
Thanks for all your feedback.

**Reply by damiensellier** [reply]
The Sony R7 is excellent, I have the DPS V77 multi-effect which combines all the Sony R series.  Gold Foil Plate is beautiful.

**Reply by dnaldoog** [reply]
They are getting pretty rare, right?

Have you tried to create a panel for one?

**Reply by damiensellier** [reply]
They are not really easy to find, I got mine from Japan from ebay. I don't think I will dive into a Panel for it because there's no sysex documentation for it unfortunately.  The manual has just the basic dumps and pre-assigned CC controls, not enough to control it without cracking the preset dumps and I don't feel going this route, it's a nightmare :)  If I find a sysex doc I'll definitely go for it.

**Reply by damiensellier** [reply]
You can get its little brother Sony HR-MP5. Almost the same algos, but with less memory so shorter delays and the in/out are unbalanced. https://youtu.be/0t7SPRjqsx4?si=uCEmjdUfv0hu8TOz&t=247

**Reply by dnaldoog** [reply]
I didn’t look into it deeply, but I thought that might be the case that the sysex was just for dumps, not actually controlling the parameters. (I am extremely slow at understanding manuals).

As I said before, I did discover what how to decode the sysex,  (8 bytes staggered to arrive at 7 bytes’ of data up to 0xFF) but maybe it’s a little bit of a dead end?

Did you ever try changing the battery? It’s a nightmare.

**Reply by damiensellier** [reply]
I didn't check the sysex format yet, I was thinking about making eventually a little panel with just the program changes and the 2 cc to control the quick controls for each programs but nothing more. It would be a nice compromise without having to retro-engineer the  algo paramaters sysex packets.

I was lucky enough to have a version with the battery socket so no need to use a soldering iron on this one. After it required a system reset. And I was good to go for the next 10 years.
Is the battery soldered in the R7? There's a retrofit part to use a CR2032 battery in a socket https://reverb.com/ca/item/38840465-sony-dps-r7-battery-replacement-clip

**Reply by damiensellier** [reply]
I have a problem with the LCD though, 2 vertical lines are missing and It seems it's repairable by applying heat on the ribbon connector with a soldering iron and a wide tip but it's not for the faint of heart. As we say, perfect is the enemy of good.

**Reply by damiensellier** [reply]
@dnaldoog do you know there's a remote control for the DPS-R7 ? It's called the RM-DPS7. Good luck to find this unobtainium  :) 
![osulufeqsknln4we6ucw](https://github.com/RomanKubiak/ctrlr/assets/55164385/59eb29a5-14e6-4f4f-af80-c1c3b6e35276)

**Comment by dnaldoog** [comment]
The battery is soldered and [it looks like I just resoldered ]( https://gearspace.com/board/so-much-gear-so-little-time/58803-solution-battery-replacement-dps-r7.html ) a new battery back in, so probably more headaches down the line 😂

**Reply by damiensellier** [reply]
I put some CR2032 sockets on several gears i have. It's really an improvement. Batteries are easy to find and it just take 3 solders to do, you'll have to solder a new battery anyway.

**Reply by damiensellier** [reply]
Next time!

**Comment by dnaldoog** [comment]
> I have a problem with the LCD though, 2 vertical lines are missing and It seems it's repairable by applying heat on the ribbon connector with a soldering iron and a wide tip but it's not for the faint of heart. As we say, perfect is the enemy of good.

_What is that saying in French? I like it._

**Reply by damiensellier** [reply]
"le mieux est l'ennemi du bien" :)

**Comment by dnaldoog** [comment]
> @dnaldoog do you know there's a remote control for the DPS-R7 ? It's called the RM-DPS7. Good luck to find this unobtainium :) 

No I didn’t know about it. Might be super rare even in Japan! Will keep a lookout for one though!
